### PR TITLE
Addresses issues #11, #17, #19, #20, #21, #24, #9, #25, #27

### PR DESCRIPTION
**Pull request by herbdool**
I've addressed these issues #11,  #17, #19, #20 #21, #24, #9, #25, #27  in this PR.

### CODE REVIEW

**Comment by herbdool** [comment]
I think most of these changes are small (though it would be good to get a second set of eyes on them before merging). Still, wondering if you @quicksketch or @biolithic wants some help merging? May also need some help merging in further work from the Drupal 7 webform since the Backdrop version is still sitting approximately 7.x-4.4

**Comment by herbdool** [comment]
Thanks @biolithic! I've closed the issues referenced. Would you be able to test and create another release too?

**Comment by biolithic** [comment]
Yes, I will test.

I don't want to be the module maintainer but I think I should at least
update community findings and sorry for the general confusion you may have
had with this module.
Thanks again!

**Comment by herbdool** [comment]
I think that's understandable @biolithic that you don't want to be the maintainer. Though I wanted to respect the general policy in backdrop-contrib and it seems that Nate has his hands full. And it's always good to have a second set of eyes on things.

The release listed on backdropcms.org has some big bugs in it which these PRs fix so it'll be good to get a new release out soon.

**Comment by biolithic** [comment]
yep, will do

**Comment by klonos** [comment]
> ...and it seems that Nate has his hands full

I just spared him a couple of minutes by trying to triage the contrib applications as best as I could, but I doubt that it will be enough :smile: 

Thanx you guys. I might be needing to use this module soon. :+1:

File: includes/webform.report.inc [hunk 1]
```diff
@@ -762,7 +762,7 @@ function webform_results_download_form_submit(&$form, &$form_state) {
  * Returns a temporary export filename.
  */
 function _webform_export_tempname() {
-  return backdrop_tempnam(config_get('webform.settings', 'webform_export_path'));
+  return backdrop_tempnam(config_get('webform.settings', 'webform_export_path'), 'webform_');
```
Comment (klonos) [inline includes/webform.report.inc:765]: shouldn't this be `backdrop_tempname` (missing an "e" at the end)

**Comment by herbdool** [comment]
Nope, no "e".
https://api.backdropcms.org/api/backdrop/core%21includes%21file.inc/function/backdrop_tempnam/1

File: includes/webform.report.inc (hunk 1 above)
Comment (herbdool) [inline includes/webform.report.inc:765]: No, it is indeed without the "e". https://api.backdropcms.org/api/backdrop/core%21includes%21file.inc/function/backdrop_tempnam/1

**Comment by klonos** [comment]
Yep, never mind me (no API expert). Just thought I should mention it just in case.

**Pull request merged**